### DISCUSSION TITLE
Launch Editor from Compare Commits?

### DISCUSSION THREAD

**Discussion by sez11a** [extrawurst/gitui#1407] [Q&A] [answered]
Stumbled upon GitUI, and I think it's great! It's missing one thing (as far as I can tell) that prevents me from using it. My use case is reviewing pull requests from others. To do that, I check out their branch, rebase master on top of it, and then launch GitUI. 

In the Log view, I tag the top commit and the last commit from master, and then hit C to compare. GitUI shows a beautiful tree on the left with only the changed files in it and a diff on the right. If I could highlight one of those files and hit Enter or E to launch my default editor (neovim), GitUI would become my only Git client. 

I'm not sure if I'm missing something in the config or if this is a feature request. Is this possible?

**Comment by extrawurst** [comment, marked as answer]
see https://github.com/extrawurst/gitui/issues/1409
